### DISCUSSION TITLE
Required location permissions on Android

### DISCUSSION THREAD

**Discussion by berhir** [shinyorg/shiny#1229] [Q&A]
Hi, I have noticed a change in the required location permissions on Android in one of the latest beta versions. I am not sure if this is a bug or the expected behavior, so I was afraid of creating a bug.

As far as I understand the [Android documentation](https://developer.android.com/training/location/permissions#foreground), requesting the foreground location permission (Allow only while using the app) should be enough if the app uses a foreground services. We used it in our app with an older alpha version of Shiny for MAUI and it worked fine. But since we updated to a newer beta version, it's required to select "Allow all the time". Otherwise, the user gets redirected to the settings page of the app.
The MAUI sample app in this repo has a similar behavior. It checks the permission already before starting the GPS tracking and shows an error "Insufficient permissions".

It seems this change was introduced with [this commit](https://github.com/shinyorg/shiny/commit/4f6475f6a729e8bb8e1d921e9c16fb18563d4aaf).

I would prefer not to ask for more permissions than required. That's why I want to ask if it's possible to change this behavior?
Thanks!

**Comment by aritchie** [comment]
This was changed recently for reasons I can't remember.  I can't promise it will stay this way as there are reasons to have the background permission still.
